readme.md

```diff
@@ -41,6 +41,28 @@ const execa = require('execa');
 })();
 ```
 
+### Running test with custom environment variable
+
+Given the following shell command:
+
+```sh
+npm test -- --json
+```
+
+Let's take a look at how we can accomplish the same using `execa`:
+
+```js
+const execa = require('execa');
+
+(async () => {
+	const options = {env: {NODE_ENV: 'production'}};
+	const {stdout, stderr} = await execa('npm', ['test', '--', '--json'], options);
+	console.log({stdout, stderr});
+})();
+```
+Notice how the arguments are specified as array elements: `['test', '--', '--json']`.
```

Comment:
There should be a blank line before this line.

Reply:
It's showing fixed under this specific commit: https://github.com/sindresorhus/execa/pull/387/commits/131bfa545482f63cac6fb4d4357e948e01379d72

But not under the main PR view 🤷‍♂ 

Maybe a github caching issue. Let me know if it's still broken for you.

Reply:
The line is here :)